pkg/pipelines/tekton/templates.go

```diff
@@ -32,6 +32,8 @@ const (
 	defaultPipelinesTargetBranch = "main"
 )
 
+var FuncRepoRef = "knative/func/main"
```

Comment:
In midstream this will be patched to `openshift-knative/kn-plugin-func/release-x.x`?